kolibri/content/models.py

```diff
@@ -3,6 +3,31 @@
 To access it, please use the public APIs in api.py
 
 The ONLY public object is ContentNode
+
+Developer note: If you modify the schema here, it has implications for the content
+import pipeline, including Kolibri Studio where the imported content databases
+are created.
+
+As such, if models or fields are added or removed, the CONTENT_SCHEMA_VERSION
+value must be incremented, with an additional constant added for the previous
+version. e.g. when the CONTENT_SCHEMA_VERSION is incremented from '2' to '3'
+a new constant VERSION_2 = '2', should be added.
+
+In addition, the new constant should be added to the mappings dict in
+./utils/channel_import.py with an appropriate ChannelImport class associated. If the
+new schema requires inference of the field when it is missing from old databases
+(i.e. it does not have a default value, or cannot be null or blank), then the
+ChannelImport classes must be updated to infer this data from old databases.
+
+A pickled SQLAlchemy schema for the new schema must also be generated using the
+generate_schema management command. This must be generated using an empty, migrated database.
+The 'version' parameter passed to the command should be the value of e.g. VERSION_2.
+All pickled schema should be registered in the CONTENT_DB_SCHEMA_VERSIONS list in
+./utils/sqlalchemybridge.py e.g. VERSION_2 should be added to the list.
```

Comment:
this process is a bit terse. expanding it a bit with concrete steps and commands to run might help?

Reply:
Sure.